app/helpers/cloud_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module CloudHelper
+  def mapzy_cloud?
+    ENV["MAPZY_CLOUD"] == "true"
+  end
+end
```

Comment:
Both `allow_registration?` and `mapzy_cloud?` can be in ApplicationHelper directly I think

Reply:
Fixed in ef8a446